server/service/src/invoice_line/inbound_shipment_line/update/generate.rs

```diff
@@ -90,7 +90,7 @@ fn generate_line(
     }
 
     update_line.total_after_tax =
-        update_line.cost_price_per_pack * update_line.number_of_packs as f64;
+        update_line.total_before_tax * (f64::from(1) + update_line.tax.unwrap_or_default() / 100.0);
```

Comment:
Thats what we discussed last time, right?

Reply:
Yup~